app/process/recovery/recovery.go

```diff
@@ -235,61 +239,68 @@ func (r Recovery) recoverEthereumTXMessage(tm encoding.TopicMessage) error {
 	return nil
 }
 
+type transferMessageKey struct {
+	TransactionId string
+	EthAddress    string
+	Amount        string
+	Fee           string
+	GasPriceWei   string
+	Erc20Address  string
+}
+
 func (r Recovery) processUnfinishedOperations() error {
-	// TODO messagesRepo.getUnprocessedMessages() <- should return all Messages whose TX ID is status INITIAL or RECOVERED
-	//unprocessed, err := r.transactionsRepo.GetSkippedOrInitialTransactionsAndMessages()
-	//if err != nil {
-	//	r.logger.Fatalf("Failed to get all unprocessed messages. Error: %s", err)
-	//	return err
-	//}
-
-	// Combine all Messages records into map({txId, amount, fee, gasPrice, etc}, []signatures)
-	// Iterate all keys
-	// 	If validator has not signed -> sign and submit
-	//for txn, txnSignatures := range unprocessed {
-	//hasSubmittedSignature, ctm := r.hasSubmittedSignature(txn, txnSignatures)
-	//
-	//if !hasSubmittedSignature {
-	//	r.logger.Infof("Validator has not yet submitted signature for Transaction with ID [%s]. Proceeding now...", txn)
-	//	// TODO
-	//	err = r.transfersService.VerifyFee(ctm)
-	//	if err != nil {
-	//		r.logger.Errorf("Fee validation failed for TX [%s]. Skipping further execution", transferMsg.TransactionId)
-	//	}
-	//
-	//	signature, err := r.transfersService.ValidateAndSignTxn(ctm)
-	//	if err != nil {
-	//		r.logger.Errorf("Failed to Validate and Sign TransactionID [%s]. Error [%s].", txn, err)
-	//	}
-	//
-	//	_, err = r.transfersService.HandleTopicSubmission(ctm, signature)
-	//	if err != nil {
-	//		return errors.New(fmt.Sprintf("Could not submit Signature [%s] to Topic [%s] - Error: [%s]", signature, r.topicID, err))
-	//	}
-	//	r.logger.Infof("Successfully Validated")
-	//}
-	//}
+	unprocessedTransactions, err := r.transactions.GetUnprocessedTransactions()
+	if err != nil {
+		r.logger.Fatalf("Failed to get all unprocessedTransactions messages. Error: %s", err)
+		return err
+	}
+
+	for _, transfer := range unprocessedTransactions {
+		weiBn, err := helper.ToBigInt(transfer.GasPriceWei)
+		if err != nil {
+			r.logger.Errorf("Could not parse GasPriceWei [%s] to a Big Integer for TX [%s]. Skipping further execution", transfer.GasPriceWei, transfer.TransactionId)
```

Comment:
This error log is false. We are not `skipping further execution`. 
See the other recovery for reference:
`Skipping recovery for TX [%s]. Could not parse gas price [%s] to bigint`